index.html

```diff
@@ -1795,34 +1795,42 @@ <h4 id="h-mediasource-detach" resource="#h-mediasource-detach"><span property="x
+          <div class="note">
+            <div class="note-title marker" aria-level="5" role="heading" id="h-note22"><span>Note</span></div>
+            <p class="">The media element looks for <a href="#media-segment">media segments</a> containing the <var>new playback position</var> in each <a href="#idl-def-sourcebuffer" class="internalDFN" data-link-type="dfn"><code>SourceBuffer</code></a> object in <code><a href="#dom-mediasource-activesourcebuffers">activeSourceBuffers</a></code>. Any position within a <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#timeranges-timeranges">TimeRange</a></code> in the current value of the <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#dom-htmlmediaelement-buffered">HTMLMediaElement.buffered</a></code> attribute has a buffered media segment for that position.</p>
+          </div>
+          <dl class="switch">
+            <dt>If <var>new playback position</var> is not in any <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#timeranges-timeranges">TimeRange</a></code> of <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#dom-htmlmediaelement-buffered">HTMLMediaElement.buffered</a></code></dt>
             <div class="note">
-              <div class="note-title marker" aria-level="5" role="heading" id="h-note22"><span>Note</span></div>
-              <p class="">If the <code><a href="#dom-readystate">readyState</a></code> attribute is <code><a href="#idl-def-ReadyState.ended">"ended"</a></code>, then the last <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#timeranges-timeranges">TimeRange</a></code> in a non-empty <code><a href="https://www.w3.org/TR/html51/semantics-embedded-content.html#dom-htmlmediaelement-buffered">HTMLMediaElement.buffered</a></code> may have an end time beyond coded frame presentation intervals in one or more of the currently selected or enabled <a href="#track-buffer">track buffers</a>. Seeks must still complete to all times within that range.</p>
+              <div class="note-title marker" aria-level="5" role="heading" id="h-note23"><span>Note</span></div>
+              <p class="">In this case, one or more of the objects in <code><a href="#dom-mediasource-activesourcebuffers">activeSourceBuffers</a></code> is missing <a href="#media-segment">media segments</a> for the <var>new playback position</var>.</p>
```

Comment:
I believe this note isn't particularly needed.  The first note above is useful, and your summary note below the Continue step seems useful as well.